src/docker/10-debianconsole/.prebuild.sh

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 set -e
 
+# FIXME Remove this when Docker supplies multi-arch base images
+if [ "$ARCH" == "arm" ]; then
+  BASE_IMAGE=debian:jessie
+  ARM_BASE_IMAGE=armhf/debian:jessie
```

Comment:
Yep, that's how it works. People often consider `armhf` == ARMv7 (e.g. all `armhf/*` images), even though RPi has an armhf ARMv6 processor.
`resin/rpi-raspbian` and `armhf/ubuntu` sgtm